apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -349,7 +358,15 @@ def from_h5ad(
     with tiledb.VFS(ctx=context.tiledb_ctx).open(input_path) as input_handle:
         logging.log_io(None, f"START  READING {input_path}")
         with _hack_patch_anndata():
-            anndata = ad.read_h5ad(_FSPathWrapper(input_handle, input_path), "r")
+            path: Path = (
+                input_path._path
+                if isinstance(input_path, _FSPathWrapper)
+                else input_path
+            )
+            anndata = ad.read_h5ad(
+                _FSPathWrapper(input_handle, path),
+                "r",
+            )
```

Comment:
This check seems weird. If we’re being passed an `_FSPathWrapper` here, that means that we’ve been passed a VFS file handle rather than a thing that we should be opening with VFS.